### PR TITLE
docs: fix broken link

### LINKED ISSUE #45072: Broken link to angular.json complete schema
### Description

Here the text https://angular.io/cli#workspace-and-project-configuration the link "complete schema" contains broken URL "https://github.com/angular/angular-cli/wiki/angular-workspace" which redirects to "https://github.com/angular/angular-cli"

### What is the affected URL?

_No response_

### Please provide the steps to reproduce the issue

_No response_

### Please provide the expected behavior vs the actual behavior you encountered

_No response_

### Please provide a screenshot if possible

_No response_

### Please provide the exception or error you saw

_No response_

### Is this a browser-specific issue? If so, please specify the device, browser, and version.

_No response_

### PR DESCRIPTION
**Pull request by AlirezaEbrahimkhani**
change "complete schema" link from broken link to "https://angular.io/guide/workspace-config#angular-workspace-configuration"

resolves angular#45072

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/cli/index.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ The [ng config](cli/config) command lets you set and retrieve configuration valu
 Note that option names in the configuration file must use [camelCase](guide/glossary#case-types), while option names supplied to commands can use either camelCase or dash-case.
 
 * See more about [Workspace Configuration](guide/workspace-config).
-* See the [complete schema](https://github.com/angular/angular-cli/wiki/angular-workspace) for `angular.json`.
+* See the [complete schema](https://angular.io/guide/workspace-config#angular-workspace-configuration) for `angular.json`.
```
Comment (alan-agius4) [inline aio/content/cli/index.md:72]: The link here, is exactly the same as the one above. I think we can drop this whole point.

File: aio/content/cli/index.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ The [ng config](cli/config) command lets you set and retrieve configuration valu
 Note that option names in the configuration file must use [camelCase](guide/glossary#case-types), while option names supplied to commands can use either camelCase or dash-case.
```
Comment (alan-agius4) [inline aio/content/cli/index.md:69]:
```suggestion
Note that option names in the configuration file must use [camelCase](guide/glossary#case-types), while option names supplied to commands must be dash-case.
```

File: aio/content/cli/index.md (hunk 1 above)
Comment (alan-agius4) [inline aio/content/cli/index.md:72]:
```suggestion

```

**Review by alan-agius4 — CHANGES_REQUESTED** [review]
Thanks for your contribution. Please see comments.

**Comment by AlirezaEbrahimkhani** [comment]
@alan-agius4 
Hi
I made the changes you mentioned.

**Review by alan-agius4 — APPROVED** [review]
LGTM thanks.

NIT:  can you drop the 1st commit since it’s redundant.

**Comment by AlirezaEbrahimkhani** [comment]
@alan-agius4 
Hi !
I have not done this before and I followed the instructions in the stackoverflow, I hope you approve !

**Comment by alan-agius4** [comment]
@AlirezaEbrahimkhani, instead of reverting can you drop the commit? Also, please avoid merging and instead use rebase.

Let me know if you need any help. Thanks

**Review by alan-agius4 — APPROVED** [review]
LGTM, thanks

**Comment by AlirezaEbrahimkhani** [comment]
@alan-agius4 
Thanks! The link you sent was very helpful and I made the changes you wanted.

**Comment by alan-agius4** [comment]
@AlirezaEbrahimkhani, glad you found it useful.

**Comment by alxhub** [comment]
This PR was merged into the repository by commit df3a075b02ae5885b707abe9b5a35626288466f0.

**Pull request closed without merging**